src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java

```diff
@@ -904,15 +904,21 @@ public ActionForward insertBONote(ActionMapping mapping, ActionForm form, HttpSe
         }
         
         // If the note text is blank, set the attachment description as the text. Otherwise, concatenate both to form the text.
-        if (StringUtils.isBlank(note.getNoteText())) {
-            note.setNoteText(iWantDocumentForm.getIWantDocument().getAttachmentDescription());
-        } else {
-            note.setNoteText(iWantDocumentForm.getIWantDocument().getAttachmentDescription() + ": " + note.getNoteText());
+        if(StringUtils.isBlank(iWantDocumentForm.getIWantDocument().getNoteTextOption())) {
```

Comment:
the formatting looks off in this block. A space  is missing between the if the open paren, and the else on line 914 could be on the line above

Reply:
updated